netplugin/plugin/netplugin.go

```diff
@@ -50,8 +51,6 @@ type NetPlugin struct {
 	PluginConfig  Config
 }
 
-const defaultPvtSubnet = 0xac130000
```

Comment:
if you are removing the default svc net,
can you add it in netmaster/netctl so that it uses this svc subnet  by default ?

Reply:
i think it does

Reply:
👍